run.js

```diff
@@ -124,13 +123,24 @@ function startWho() {
   });
 }
 
-function prepareMainSettings() {
+function checkGvisor() {
+  const flags = process.env.GVISOR_FLAGS || '';
+  try {
+    console.log(`Checking gvisor... flags ${flags}`);
+    child_process.execSync(`runsc --network none ${flags} do true`, { encoding: 'utf-8' });
+    console.log('gvisor ok');
+  } catch (e) {
+    console.log('gvisor FAILED');     // Stderr is already reported above, so don't repeat it.
+    throw new Error("gvisor failed; consider a different GRIST_SANDBOX_FLAVOR");
+  }
+}
 
-  // Enable sandboxing by default.
-  setDefaultEnv('GRIST_SANDBOX_FLAVOR', 'gvisor');
-  // TODO: gvisor may fail on some old hardware, or in environments with
-  // particular limits. It would be kind to catch that and give clear
-  // feedback to installer.
+function prepareMainSettings() {
+  if (process.env.GRIST_SANDBOX_FLAVOR === 'gvisor') {
```

Comment:
What do you think about doing this in `grist-core`? A quick smoke check of `runsc` seems useful outside of the context of `grist-omnibus`. An advantage of doing it here is the failure is very early, I do see that...

Reply:
Moved to grist-core. On the side of grist-omnibus, made it exit if the server exits, and did it for all the servers that should be long-running since I think that's better.